app/util/process_utils.py

```diff
@@ -23,3 +25,26 @@ def kill_gracefully(process, timeout=2):
         stdout, stderr = process.communicate()
 
     return process.returncode, stdout, stderr
+
+
+def Popen_with_delayed_expansion(cmd, *args, **kwargs):
```

Comment:
Probably not patching. We'd probably enforce via convention (lightest weight), custom lint rule, or unit test blacklist.

I'm not too concerned about the enforcing since we very rarely add Popen calls. The main thing I was wondering is if we should just name this Popen.